### PR TITLE
Jaad

### PR DESCRIPTION
**Pull request by JaadVdW**
Larger update:
- changed the html on the ratings page to accommodate a view IMDB page button and a view trailer button
- added the functionality for these buttons
     - this required declaring a new variable similarly to how MovieID was done, but for movie_name
     - rafiq this needs to be set in your method, right now it is just using the default, again same as movieID
- the code for the new JS function that will handle both YES and NO is there
- endpoint for this new function is also in the python file
- these are not active yet, sticking with the original functionality we had tested but they are ready to go and should work

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -63,7 +63,9 @@ async function create_user(){
     window.location.replace(redirect);
 }
 
+// declaring global varibles to be used across functions
 var activeMovieID = 1;
+var activeMovieName = "Knives Out";
```
Comment (andreimastin) [inline app.js:68]: I assume these are placeholder values that get over written?

File: app.js [hunk 2]
```diff
@@ -131,35 +134,80 @@ async function clickedYes() {
...
+//function to pass info to backend when user clicks they dislike the displayed movie
 async function clickedNo() {
+    //Grab the active userID from URL
+    var vars = {};
+    var parts = window.location.href.replace(/[?&]+([^=&]+)=([^&]*)/gi, function (m, key, value) {
+        vars[key] = value;
+    });
+    //log IDs
+    console.log(vars.id);
+    console.log(activeMovieID);
```
Comment (andreimastin) [inline app.js:151]: Looks good

File: application.py [hunk 3]
```diff
@@ -376,24 +376,40 @@ def rate_yes():
...
+     db.session.commit()
+     # call method to display movie, will display a new unseen movie
 
-#     return ({'response':'Good'})
-#     # this method will insert into the user_rates_movie table 
+     return ({'response':'Good'})
+     # this method will insert into the user_rates_movie table 
 
+# ENDPOINT - User rates movie
+# This is an endpoint to consolidate the Yes and No separate endpoints, it will handle both
```
Comment (andreimastin) [inline application.py:395]: This is good work using consolidating the yes and no endpoints with a single method.

File: application.py [hunk 4]
```diff
@@ -376,24 +376,40 @@ def rate_yes():
...
+     # call method to display movie, will display a new unseen movie
+
+     return ({'response':'Good'})
+     # this method will insert into the user_rates_movie table 
 
 # Run server
 if __name__ == '__main__':
 
-    application.run(host='0.0.0.0')
-    #application.run(debug=True)
```
Comment (andreimastin) [inline application.py]: Make Edit to this again>

File: rating.html [hunk 5]
```diff
@@ -95,16 +89,26 @@ <h5 class="subtitle is-5" style="text-align: center; padding-bottom: 0px !import
...
                      </td>
-                 </tr>
 
+                     <td>
+                         <input type="button" class="Trailer Button" value="View Trailer" onClick="viewTrailer()" style="font-size:30px;font-weight:bold;border-color: black; color:white; background-color: blue; width:300px">
+                     </td>
+                 </tr>
 
-                     <!--The OnClick() values for the buttons MUST be changed to accurate ones-->
+                 <!--The OnClick() values for the buttons MUST be changed to accurate ones-->
```
Comment (andreimastin) [inline rating.html]: These are accurate now, this comment can be removed.

**Review by andreimastin — COMMENTED** [review]
Looks good Jaad, lots of progress made, You may want to consider commenting on the endpoints and methods that are not being used yet but that are ready to go. Also, make sure to edit server host before submitting.

File: app.js (hunk 1 above)
Comment (JaadVdW) [reply app.js:68]: These are where we create the variables for their use, the values they take are essentially place holders. They will change within Rafiq's method when it updates the page to show a new movie

**Comment by JaadVdW** [comment]
Alright made tweaks to the files, should be good now. Will add more comments when we switch to using the other methods

**Pull request merged**